README.md

```diff
@@ -23,6 +23,29 @@ After you've defined your DB configuration in the `.env.{development,test}.local
 * `bin/rails db:prepare` (for the development database)
 * `RAILS_ENV=test bin/rails db:prepare` (for the test database)
 
+**ElasticMQ**
+
+Technically not required to setup or have this running just yet, but might be neccessary for some functionality.  
+If interested, go ahead and expand this section.  
+
+<details>
+<summary>Run ElasticMQ and queue processor</summary>
+
+The datastore requires for some functionality a message queue. This message queue is AWS SQS on cloud-deployed 
+environments (i.e. kubernetes) but it is not practical to use AWS SQS for local development.
+
+[ElasticMQ](https://github.com/softwaremill/elasticmq) is used instead, as an in-memory message queue with an 
```

Comment:
I think this is good - if we ever need to emulate more we could look at https://github.com/localstack/localstack but I think for now ElasticMQ is the best approach